pages/common/cat.md

```diff
@@ -17,3 +17,11 @@
 - Number all output lines:
 
 `cat -n {{file}}`
+
+- Display "$" at end of each line:
+
+`cat -E {{file}}`
+
+- Display TAB characters as ^I:
```

Comment:
What is the usefulness of this ? I can still understand why `-E` may be useful. But `-T` just seems like a corner case to me.